src/main/java/org/example/springproject/exception/CustomGlobalExceptionHandler.java

```diff
@@ -46,6 +47,15 @@ public ResponseEntity<Object> handleEntityNotFoundExceptions(EntityNotFoundExcep
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(NoSuchElementException.class)
```

Comment:
Do you need this exception? Of it's the same as EntityNotFoundException, leave only one